app/helpers/contacts_helper.rb

```diff
@@ -9,4 +9,11 @@ def start_a_conversation_link(aspect, contacts_size)
                   title: t("contacts.index.start_a_conversation")
     end
   end
+
+  def gon_load_contact(contact)
+    Gon.preloads[:contacts_collection] ||= []
+    if contact.id.nil? || !Gon.preloads[:contacts_collection].map {|contact| contact[:id] }.include?(contact.id)
```

Comment:
> I'm also not too found of things like collection, map, array, hash in the name of things, can we just call it contacts or user_contacts?

Then I'll have to rename [this preload](https://github.com/diaspora/diaspora/blob/develop/app/controllers/people_controller.rb#L81-L83) to, perhaps, `contact_count`. Are you ok with that?